engine/apps/api/serializers/organization.py

```diff
@@ -46,15 +46,18 @@ class Meta:
             "grafana_incident_enabled",
         ]
 
-    def get_slack_channel(self, obj):
+    def get_slack_channel(self, obj: Organization):
         from apps.slack.models import SlackChannel
 
-        if obj.general_log_channel_id is None or obj.slack_team_identity is None:
+        org_general_log_channel_id = obj.general_log_slack_channel_slack_id
+        slack_team_identity = obj.slack_team_identity
+
+        if org_general_log_channel_id is None or slack_team_identity is None:
             return None
         try:
-            channel = obj.slack_team_identity.get_cached_channels().get(slack_id=obj.general_log_channel_id)
+            channel = slack_team_identity.get_cached_channels().get(slack_id=org_general_log_channel_id)
```

Comment:
can we do `channel = obj.general_log_slack_channel` here instead of `slack_team_identity.get_cached_channels().get`? 🤔

Reply:
updated in https://github.com/grafana/oncall/pull/5191/commits/ce5a0978ccc151dd9b87a32b027a4ce8bf84fb79